Tests/Unit/Model/IdealPaymentMethodTest.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+use Wirecard\Oxid\Core\PaymentMethodHelper;
+use Wirecard\Oxid\Model\IdealPaymentMethod;
+use Wirecard\Oxid\Model\SepaCreditTransferPaymentMethod;
+
+use Wirecard\PaymentSdk\Config\Config;
+use Wirecard\PaymentSdk\Config\PaymentMethodConfig;
+use Wirecard\PaymentSdk\Transaction\IdealTransaction;
+
+use OxidEsales\Eshop\Core\Registry;
+
+class IdealPaymentMethodTest extends OxidEsales\TestingLibrary\UnitTestCase
+{
+    /**
+     * @var IdealPaymentMethod
+     */
+    private $oPaymentMethod;
```

Comment:
can you name it $_oPaymentMethod with an underscore to have our guidelines applied!?

Reply:
:+1: